src/ch/ehi/ili2db/gui/Config.java

```diff
@@ -830,4 +831,12 @@ public String getSqlExtRefCols() {
     public void setSqlExtRefCols(String value) {
         setValue(SQL_EXTREF,value);
     }
+
```

Comment:
so wird die Config in der DB persistiert. Es ist ja aber eine Option die je Import unterschiedlich gewählt werden kann. Darum besser analog wie disableRounding implementieren